### ISSUE TITLE
add no lhr card Eth+Alph support

### ISSUE THREAD

**Issue by xmhlg** [trexminer/T-Rex#1080]
Dear Dev Team,
Now, when we have working support for Eth+Alph mining, the next, most urgent feature should be to combine Eth+ton 
Or add no lhr card Eth+Alph support。
thk！

**Comment by trexminer** [comment]
non-LHR support is added in https://github.com/trexminer/T-Rex/releases/tag/0.25.6
